ansible_bender/__main__.py

```diff
@@ -0,0 +1,5 @@
+from .cli import main
```

Comment:
I'd add a module docstring too. Smth like
```suggestion
"""Ansible Bender CLI entry point."""
from .cli import main
```
but that's minor.